code/go/0chain.net/blobbercore/handler/grpc_helpers.go

```diff
@@ -83,3 +83,10 @@ func CustomMatcher(key string) (string, bool) {
 		return runtime.DefaultHeaderMatcher(key)
 	}
 }
+
+func uploadHandler(w http.ResponseWriter, r *http.Request, pathParams map[string]string) {
```

Comment:
No need for this actually

Reply:
where should this handler be implemented?

Reply:
No actually we already have this implemented, we can just use that. See the `UploadHandler` in the handler.go file.